rpms/pulp/pulp.spec

```diff
@@ -0,0 +1,853 @@
+
+install -pm644 server/usr/share/pulp/templates/* %{buildroot}/%{_datadir}/pulp/templates/
+
+# pulp-server installation
+install -d %{buildroot}%{_sysconfdir}/%{name}/content/sources/conf.d
+install -d %{buildroot}%{_sysconfdir}/%{name}/server/plugins.conf.d
+install -d %{buildroot}%{_sysconfdir}/%{name}/vhosts80
+install -d %{buildroot}%{_sysconfdir}/default/
+install -d %{buildroot}%{_usr}/lib/tmpfiles.d/
+install -d %{buildroot}%{_datadir}/pulp/selinux/server
+install -d %{buildroot}%{_var}/cache/%{name}
+install -d %{buildroot}%{_var}/run/%{name}
+install -d %{buildroot}%{_sharedstatedir}/rpm-state/pulp/old-version
+install -d %{buildroot}%{_sharedstatedir}/%{name}/content
+install -d %{buildroot}%{_sharedstatedir}/%{name}/published
+install -d %{buildroot}%{_sharedstatedir}/%{name}/static
+install -d %{buildroot}%{_sharedstatedir}/%{name}/templates
+install -d %{buildroot}%{_sharedstatedir}/%{name}/uploads
+install -d %{buildroot}%{_var}/www
+
+# Install SELinux policy modules
+pushd server/selinux/server
+./install.sh %{buildroot}%{_datadir}
+# Remove the unneeded .if files
+rm %{buildroot}%{_datadir}/selinux/devel/include/apps/pulp-*.if
+popd
+
+install -pm755 server/bin/* %{buildroot}/%{_bindir}
+install -pm644 server/etc/default/systemd_pulp_celerybeat \
+    %{buildroot}/%{_sysconfdir}/default/pulp_celerybeat
+install -pm644 server/etc/default/systemd_pulp_resource_manager \
+    %{buildroot}/%{_sysconfdir}/default/pulp_resource_manager
+install -pm644 server/etc/default/systemd_pulp_workers \
+    %{buildroot}/%{_sysconfdir}/default/pulp_workers
+install -pm644 server/etc/httpd/conf.d/pulp_apache_24.conf \
+    %{buildroot}/%{_sysconfdir}/httpd/conf.d/pulp.conf
+install -pm644 server/etc/httpd/conf.d/pulp_content.conf \
+    %{buildroot}/%{_sysconfdir}/httpd/conf.d/pulp_content.conf
+install -pm644 server/etc/pulp/* %{buildroot}/%{_sysconfdir}/%{name}
+install -pm644 server/%{_unitdir}/* %{buildroot}/%{_unitdir}
+install -pm644 server/usr/lib/tmpfiles.d/* %{buildroot}/%{_usr}/lib/tmpfiles.d/
+install -pm755 server/selinux/server/enable.sh %{buildroot}%{_datadir}/pulp/selinux/server
+install -pm755 server/selinux/server/uninstall.sh %{buildroot}%{_datadir}/pulp/selinux/server
+install -pm755 server/selinux/server/relabel.sh %{buildroot}%{_datadir}/pulp/selinux/server
+install -pm644 server/usr/share/pulp/wsgi/* %{buildroot}/%{_datadir}/pulp/wsgi/
+
+# Web Content
+ln -s %{_sysconfdir}/pki/pulp/rsa_pub.key %{buildroot}/%{_var}/lib/pulp/static
+ln -s %{_var}/lib/pulp/published %{buildroot}/%{_var}/www/pub
+
+# These files will be generated during installation
+touch %{buildroot}/%{_sysconfdir}/pki/%{name}/ca.crt
+touch %{buildroot}/%{_sysconfdir}/pki/%{name}/ca.key
+touch %{buildroot}/%{_sysconfdir}/pki/%{name}/rsa.key
+touch %{buildroot}/%{_sysconfdir}/pki/%{name}/rsa_pub.key
+
+# pulp-streamer installation
+install -d %{buildroot}/%{_sysconfdir}/default/
+install -d %{buildroot}/%{_sysconfdir}/httpd/conf.d/
```

Comment:
It looks like this got made earlier as part of the "multiple sub-packages use these dirs" section. Doesn't hurt anything to be here, but it should be able to go as well.